pages/blog/swift-graphql.mdx

```diff
@@ -0,0 +1,160 @@
+import withArticle from '../../ui/blog/article';
+
+<!-- Configuration -->
+
+export default withArticle({
+  title: 'SwiftGraphQL - A GraphQL client that lets you forget about GraphQL',
+  author: 'maticzav',
+  tags: ['graphql', 'swift', 'client', 'swift-graphql'],
+  date: '2021-03-25',
+  description: 'An introduction article for SwiftGraphQL client.',
+  image:
+    'https://github.com/the-guild-org/oneblog/blob/master/img/704/1*emrpS_BYaLltTGadD6GaIw.png?raw=true',
```

Comment:
Could you ask Idan on our Slack for get you a nice image?

Reply:
I like the one Idan made, I think you were thinking of something similar, right?